### PR TITLE
Revise compensation doc to reflect bisq >1.0

### PR DESCRIPTION
**Pull request by m52go**

### CODE REVIEW

**Comment by m52go** [comment]
@ripcurlx @devinbileck if one of you could take a quick look at this (as well as https://github.com/bisq-network/compensation/pull/288) so we can merge soon, would be great. People are making compensation requests without good guidelines.

File: compensation.adoc [hunk 1]
```diff
@@ -2,79 +2,103 @@
...
-It's important to make use of this time so that you can make a well-informed vote on everyone's work.
+BSQ is issued on Bisq when a compensation request is approved through DAO voting, so your compensation request needs to be filed there in order for you to actually be paid.
 
-=== Vote
+Once you've documented all details of your request in a new issue on GitHub, make a new compensation request proposal on Bisq:
+
+.Make a new compensation request in the Bisq DAO.
+image::make-compensation-request.png[Make a new compensation request in the Bisq DAO]
+
+Make sure you select `Compensation request` as the proposal type. Also makre sure you use a name that stakeholders will recognize (or at least one they can cross-reference with your other online profiles like GitHub, Slack, forum, etc).
```
Comment (devinbileck) [inline compensation.adoc]: Typo `makre`

File: compensation.adoc [hunk 2]
```diff
@@ -2,79 +2,103 @@
...
+
+If you need to make a change, you can delete the proposal and make a new one while the proposal phase is still active, but you'll need to pay the proposal fee again. Once the proposal phase is over, proposals cannot be added or removed until the next proposal phase.
+
+NOTE: A compensation request is a type of proposal, and proposals cost a fee to make in the Bisq DAO. Currently that fee is 2 BSQ. Since a compensation request is intended to issue new BSQ, you'll also need to include a small amount of bitcoin to be colored into BSQ if your proposal is accepted—100 satoshis × [requested BSQ amount]. See more details in our https://docs.bisq.network/dao-user-reference.html#proposal-phase[User Guide].
+
+The BSQ and BTC is automatically added to your proposal transation by Bisq, but the BTC needs to be in your Bisq BTC wallet. Make sure you have enough in there before making your request. Keep in mind that you'll also need some BSQ and BTC for the other proposal phases (vote & reveal vote).
```
Comment (devinbileck) [inline compensation.adoc]: vote reveal should not require BSQ/BTC?

File: compensation.adoc [hunk 3]
```diff
@@ -2,79 +2,103 @@
...
+
+When you successfully submit your proposal in the DAO, it'll propagate across the Bisq peer-to-peer network and be ready for stakeholders to vote on in the voting phase. If your request is approved, you will see the BSQ you requested in your wallet after the voting cycle is over.
```
Comment (devinbileck) [inline compensation.adoc]: > after the voting **phase** is over

File: compensation.adoc (hunk 2 above)
Comment (m52go) [reply compensation.adoc]: Vote reveal is a bitcoin transaction, so it requires BTC but not BSQ, I believe.

Comment (devinbileck) [reply compensation.adoc]: Ah, correct. BTC mining fee only.

**Pull request merged**